pkg/stanza/pipeline/directed.go

```diff
@@ -26,8 +26,9 @@ var alreadyStopped = stanzaerrors.NewError("pipeline already stopped", "")
 
 // DirectedPipeline is a pipeline backed by a directed graph
 type DirectedPipeline struct {
-	Graph *simple.DirectedGraph
+	// Deprecated [v0.100.0]
 	*zap.SugaredLogger
```

Comment:
We are not super consistent with deprecation of this vs outright removal. Is there any reason for it?

Reply:
I tried to make an effort to minimize breaking changes but I think in a few cases leaving the logger in place could be less clear since it would just be unused. Are there are any instances in particular you think I should revisit?

Reply:
I think I misread this, because I can't find said instances now :smile: Marking as resolved